final-project/Figures/api-component.mmd

```diff
@@ -0,0 +1,20 @@
+C4Container
+title Component diagram for APIProxy for OpenTogetherTube
```

Comment:
This diagram doesn't seem to have anything to do with the HTTP API reverse proxy that the balancer will have.

Reply:
This is not fixed.